### ISSUE TITLE
Look for potential useful commands that can be added to asisstant

### ISSUE THREAD

**Issue by orzklv** [mad-maids/telegram#3]
Except warning and etc, we need to add more commands that can be useful to engage our community.

**Comment by akumarujon** [comment]
I was thinking about to create a function to send memes to `memes` topic. Like `meme time`, It automatically sends memes every day.

Looking for meme websites(in order to get api or parse) or meme API's.

**Comment by akumarujon** [comment]
After some time, I thought about it.

Is this bot running on Deno? If so we could use [Deno.cron](https://docs.deno.com/deploy/kv/manual/cron) for sending memes.

**Comment by orzklv** [comment]
> Is this bot running on Deno? 
Yes, I don't have plans to RIIR anything here as I see Mad Maids sort of lightweight community for geeks.

> If so we could use [Deno.cron](https://docs.deno.com/deploy/kv/manual/cron) for sending memes.
PRs are welcomed 😁

**Comment by akumarujon** [comment]
It works, I created another bot for testing purposes.

The code looks like:
![imgae](https://i.imgur.com/ltBUdIy.png)

If you want to check you can join [this group](https://t.me/+vk2UHR1Lx7hjMDVi)
and I'm using https://github.com/deep5050/programming-memes for getting memes from

**Comment by orzklv** [comment]
Wouldn't be better if we forked this repo and continued adding more memes and also maintain this thing

**Comment by akumarujon** [comment]
yes, it is probably a better idea
